misc/kafka-util/README.md

```diff
@@ -0,0 +1,3 @@
+# Kafka Utilities
```

Comment:
There is also a `src/kafka-util` directory that we could use (though maybe we need Python code to be in `misc`, for whatever reason? If so, this is fine.)

Reply:
Yes, Python code needs to live in `misc` for it to get picked up by CI etc. Even if we did have a less backwater place to put it, mixing Python and Rust code in the same directories is going to be more trouble than it's worth.

Reply:
Oh, I take that back; it's an mzbuild image. Shrug. It's unrelated to the code in `src/kafka-util` so putting it here makes sense.

Reply:
Do we feel strongly about this being in Rust vs Python? We can file a follow up ticket and I can port this to Rust if that's important.

Reply:
I think Python is fine. I wrote the other `kafka-util` stuff in Rust simply because it's the language I'm currently most familiar with, but the only really important reason to use Rust for something is if it needs to interoperate with the current Rust codebase, which this doesn't.